schema/sql/schema.sql

```diff
@@ -97,6 +97,14 @@ create table "ssh_public_key" (
   "uid" integer references passwd (uid) on delete cascade
 );
 
+create domain pgp_sha256_fingerprint bytea check (length(value) = 64);
+
+create table "gpg_public_key" (
+  "uid" integer references passwd (uid) on delete cascade,
+  "fingerprint" pgp_sha256_fingerprint not null,
```

Comment:
can't we derive this rather than storing it? pgcrypto has a `pgp_key_id` function.... what does that do?

Reply:
When trying to use `pgp_key_id` against my full public key, I got an error that it doesn't support multiple keys. when trying to use `pgp_key_id` against the "forced" export (using a `!` on the end of the key), I got an error that "no encryption key found".

Reply:
Since this is broken currently, we'll be dropping the field